File: src/main/java/top/zbeboy/isy/service/graduate/design/GraduationDesignDatumTypeService.java
package top.zbeboy.isy.service.graduate.design;

import top.zbeboy.isy.domain.tables.pojos.GraduationDesignDatumType;

import java.util.List;

/**
 * Created by zbeboy on 2017/6/23.
 */
public interface GraduationDesignDatumTypeService {
    /**
     * 查询全部类型
     *
     * @return 全部
     */
    List<GraduationDesignDatumType> findAll();
}

File: src/main/java/top/zbeboy/isy/service/graduate/design/GraduationDesignDatumTypeService.kt
package top.zbeboy.isy.service.graduate.design

import top.zbeboy.isy.domain.tables.pojos.GraduationDesignDatumType

/**
 * Created by zbeboy 2018-01-29 .
 **/
interface GraduationDesignDatumTypeService {
    /**
     * 查询全部类型
     *
     * @return 全部
     */
    fun findAll(): List<GraduationDesignDatumType>
}